cpp/ql/src/semmle/code/cpp/models/implementations/StdSet.qll

```diff
@@ -132,9 +132,8 @@ private class StdSetErase extends TaintFunction {
  */
 private class StdSetEqualRange extends TaintFunction {
   StdSetEqualRange() {
-    this
-        .hasQualifiedName("std", ["set", "unordered_set"],
-          ["lower_bound", "upper_bound", "equal_range"])
+    this.hasQualifiedName("std", ["set", "unordered_set"],
+      ["lower_bound", "upper_bound", "equal_range"])
```

Comment:
I'm surprised by this change. I would have expected the second line to be indented two levels more. If `hasQualifiedName` has a result, could we then get

```ql
    this.hasQualifiedName("std", ["set", "unordered_set"],
      ["lower_bound", "upper_bound", "equal_range"])
        .hasSomeProperty()
```

That doesn't seem right. But perhaps it's not an issue in practice.

Reply:
Yeah, that's actually a reasonable expectation, which I think makes perfect sense. Currently the indentation block isn't started until after printing the call, but it would make more sense to start it before, which would have this effect. It does require some refactoring of the autoformatter code, so I'll make a note of this and leave it for future work if that's ok.